blog/2022/12-19-from-cloud2sql/index.md

```diff
@@ -0,0 +1,141 @@
+---
+authors: [lukas]
+tags: [cloud2sql, graph, SQL]
+image: ./img/banner-social.png
+---
+
+# From cloud2sql: integrating cloud data into existing SQL workflows
+
+```mdx-code-block
+import ZoomPanPinch from '@site/src/components/ZoomPanPinch';
+```
+
+![Turning graph data into tables](./img/banner.png)
+
+One of the key features of Resoto is its ability to collect data from a wide range of cloud providers, including Amazon Web Services (AWS), DigitalOcean, and Google Cloud Platform (GCP). This makes it easy to get a comprehensive view of your cloud infrastructure, no matter where it is deployed.
+
+But what sets Resoto apart from other cloud data collection tools is its ability to enrich the data it collects and make additional connections. This means that Resoto not only gathers raw data about your cloud resources, but also adds additional context and information that can help you better understand your cloud environment.
+
+<!--truncate-->
+
+For example, when collecting data about Elastic Block Store (EBS) volumes in AWS, Resoto uses Amazon CloudWatch time series data to find out when a volume had its last read and write IO. This information is then used to enrich the raw volume data with "last_access" and "last_update" timestamp properties, which makes it easy to answer when a particular EBS volume was last used, not just that it is not in use right now.
+
+Resoto also makes additional connections, for instance between your K8S clusters and the underlying cloud provider systems. So you can easily see how your Kubernetes pods are related to the virtual machines that they are running on, and how those virtual machines are related to the storage volumes that they are using. This can provide valuable insights into the structure and organization of your cloud environment, and can help you make better decisions about how to manage and optimize it.
+
+The data collected by Resoto is complex and deeply nested, with many levels of dependencies between different cloud resources. To manage this complexity, Resoto uses a graph data model where each cloud resource is represented by a node in the graph with edges connecting dependent nodes.
+
+Let us take a closer look at all the dependencies of an AWS Elastic Load Balancer (ELB) named `aws_elb` in the model:
+
+![Dependencies of an AWS ELB](./img/aws_elb_relationships.svg)
```

Comment:
While this diagram isn't as complex, should wrap with the `<ZoomPanPinch>` for consistency. Users viewing at small screen widths might also appreciate the ability to zoom in.